stl/src/cthread.cpp

```diff
@@ -99,9 +99,16 @@ _Thrd_id_t _Thrd_id() { // return unique id for current thread
 }
 
 unsigned int _Thrd_hardware_concurrency() { // return number of processors
-    SYSTEM_INFO info;
-    GetNativeSystemInfo(&info);
-    return info.dwNumberOfProcessors;
+    DWORD_PTR process_affinity;
+    DWORD_PTR system_affinity;
+    if (GetProcessAffinityMask(GetCurrentProcess(), &process_affinity, &system_affinity))

```

Comment:
Our conventions require braces for all control flow. Consistently following this rule prevents introducing mistakes during maintenance.